src/sentry/models/integrations/utils.py

```diff
@@ -15,7 +15,7 @@
 
 
 def get_provider(instance: Integration | RpcIntegration) -> IntegrationProvider:
-    from sentry import integrations
+    from sentry.integrations.manager import default_manager as integrations
 
     return integrations.get(instance.provider)
```

Comment:
Probably a sign that this method doesn't belong here, since the import should be happening the other way around. Or, if anything, move this to a util file under manager